tslint.json

```diff
@@ -103,7 +104,8 @@
             "onlyDependOnLibsWithTags": [
               "scope:shared",
```

Comment:
scope:shared or "scope:pmp-web-shared"?

Reply:
Basically both. I'm not sure if it should be here before it is necessary.